tests/integration/framework/process/placement/placement.go

```diff
@@ -164,3 +175,123 @@ func (p *Placement) InitialClusterPorts() []int {
 func (p *Placement) CurrentActorsAPILevel() int {
 	return 20 // Defined in pkg/actors/internal/api_level.go
 }
+
+func (p *Placement) EstablishConn(ctx context.Context) (*grpc.ClientConn, error) {
+	return grpc.DialContext(ctx, "localhost:"+strconv.Itoa(p.port),
+		grpc.WithBlock(),
+		grpc.WithTransportCredentials(grpcinsecure.NewCredentials()),
+	)
+}
+
+func RegisterHost(t *testing.T, ctx context.Context, conn *grpc.ClientConn, msg *placementv1pb.Host, placementMessage chan any, stopCh chan struct{}) {
```

Comment:
Please do not pass in channels into this function which is used to signal. This function should be completely self contained and only return on completion of the message.

We can use a `require` inside this function to ensure that the message has been sucessful.